{"name": "Check right information", "status": "broken", "statusDetails": {"message": "SystemExit: Testing failed in account overview with credentials : test3@test.com & test345", "trace": "self = <Web_Testing.Pages.AccountOverviewPage.AccountOverviewPage object at 0x03CFA9A0>, email = 'test3@test.com', password = 'test345'\ndob = <Web_Testing.helperClasses.DOB object at 0x04209BE0>, username = 'Testing TeamX'\n\n    def account_overview_check(self, email, password, dob, username):\n        try:\n>           if self.initialize_account_overview():\n\nWeb_Testing\\Pages\\AccountOverviewPage.py:124: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <Web_Testing.Pages.AccountOverviewPage.AccountOverviewPage object at 0x03CFA9A0>\n\n    def initialize_account_overview(self):\n        try:\n            username = self.find_elements_by_xpath(\n                \"/html/body/div[1]/div/div/div/div[2]/div[2]/div/div[2]/div/div[3]/div/div[1]/div[2]\")\n            email = self.find_elements_by_xpath(\n                \"/html/body/div[1]/div/div/div/div[2]/div[2]/div/div[2]/div/div[3]/div/div[2]/div[2]\")\n            dob = self.find_elements_by_xpath(\n                \"/html/body/div[1]/div/div/div/div[2]/div[2]/div/div[2]/div/div[3]/div/div[2]/div[2]\")\n            country = self.find_elements_by_xpath(\n                \"//*[@id='root']/div/div/div/div[2]/div/div/div[2]/div/div[4]/div[2]/h5\")\n            if username is not None:\n>               self.account_overview_username = username.text\nE               AttributeError: 'list' object has no attribute 'text'\n\nWeb_Testing\\Pages\\AccountOverviewPage.py:86: AttributeError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <Web_Testing.Tests.test_accountoverview.TestAccountOverview object at 0x04209580>, setup_check_information = None\n\n    @allure.severity(allure.severity_level.CRITICAL)\n    @allure.story(\"Account Overview Tests\")\n    @allure.sub_suite(\"Check right information\")\n    @allure.title(\"Check right information\")\n    @allure.description(\"Check user information and signing in with the following credentials email : \"\n                        \"test_projectX@hotmail.com & password: \")\n    @pytest.mark.Do\n    @pytest.mark.AccountOverview\n    def test_case_1(self, setup_check_information):\n        self.account_overview_page.click_account()\n        time.sleep(3)\n>       if self.account_overview_page.account_overview_check(\"test3@test.com\",\n                                                        ConstantsClass().get_pass(\"test3@test.com\"),\n                                                        ConstantsClass().get_dob(\"test3@test.com\"),\n                                                        ConstantsClass().get_name(\"test3@test.com\")):\n\nWeb_Testing\\Tests\\test_accountoverview.py:70: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nWeb_Testing\\Pages\\AccountOverviewPage.py:139: in account_overview_check\n    exit('Testing failed in account overview with credentials : ' + email + \" & \" + password)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = Use exit() or Ctrl-Z plus Return to exit, code = 'Testing failed in account overview with credentials : test3@test.com & test345'\n\n    def __call__(self, code=None):\n        # Shells like IDLE catch the SystemExit, but listen when their\n        # stdin wrapper is closed.\n        try:\n            sys.stdin.close()\n        except:\n            pass\n>       raise SystemExit(code)\nE       SystemExit: Testing failed in account overview with credentials : test3@test.com & test345\n\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\_sitebuiltins.py:26: SystemExit"}, "description": "Check user information and signing in with the following credentials email : test_projectX@hotmail.com & password: ", "start": 1586104101917, "stop": 1586104104991, "uuid": "1f4fa070-672c-468e-8204-666a31649473", "historyId": "c38811e362e09239520d458ec3ff2c89", "testCaseId": "f8495a297c2ea54805288fbb7cefa284", "fullName": "Web_Testing.Tests.test_accountoverview.TestAccountOverview#test_case_1", "labels": [{"name": "story", "value": "Account Overview Tests"}, {"name": "feature", "value": "Account Overview Tests"}, {"name": "parentSuite", "value": "End to End testing"}, {"name": "suite", "value": "Account Overview Tests"}, {"name": "severity", "value": "critical"}, {"name": "subSuite", "value": "Check right information"}, {"name": "tag", "value": "AccountOverview"}, {"name": "tag", "value": "Do"}, {"name": "host", "value": "Eman"}, {"name": "thread", "value": "36688-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Web_Testing.Tests.test_accountoverview"}]}